batch/extension-jberet/src/main/java/org/wildfly/extension/batch/jberet/deployment/JobOperationUpdateStepHandler.java

```diff
@@ -34,6 +34,14 @@
  * @author <a href="mailto:jperkins@redhat.com">James R. Perkins</a>
  */
 abstract class JobOperationUpdateStepHandler extends JobOperationStepHandler {
```

Comment:
This class is not well named now. It's name led me to dig into this PR quite a bit because I was concerned an "update step handler" would not only be used for read-only ops. But it is, so that's good. :)

You should probably rename it JobOperationReadOnlyStepHandler since in the future someone else may get confused and extend it for a write op.